openapi-specs/nomis-sync-api-docs.json

```diff
@@ -1,9510 +1 @@
-      },
-      "NonAssociationResponse": {
-        "required": [
-          "effectiveDate",
-          "nsOffenderNo",
-          "offenderNo",
-          "reason",
-          "recipReason",
-          "type",
-          "typeSequence",
-          "updatedBy"
-        ],
-        "type": "object",
-        "properties": {
-          "offenderNo": {
-            "type": "string",
-            "description": "Noms id of the prisoner",
-            "example": "A1234DF"
-          },
-          "nsOffenderNo": {
-            "type": "string",
-            "description": "Noms id of the other prisoner",
-            "example": "A1234EG"
-          },
-          "typeSequence": {
-            "type": "integer",
-            "description": "Sequence number",
-            "format": "int32",
-            "example": 1
-          },
-          "reason": {
-            "type": "string",
-            "description": "Reason code of the first prisoner, domain NON_ASSO_RSN",
-            "example": "VIC"
-          },
-          "recipReason": {
-            "type": "string",
-            "description": "Reason code of the other prisoner, domain NON_ASSO_RSN",
-            "example": "PER"
-          },
-          "type": {
-            "type": "string",
-            "description": "Type code, domain NON_ASSO_TYP",
-            "example": "WING"
-          },
-          "authorisedBy": {
-            "type": "string",
-            "description": "Free text name of staff member",
-            "example": "Joe Bloggs"
-          },
-          "updatedBy": {
-            "type": "string",
-            "description": "Last updated by",
-            "example": "JSMITH_GEN"
-          },
-          "effectiveDate": {
-            "type": "string",
-            "description": "Effective date",
-            "format": "date",
-            "example": "2022-08-12"
-          },
-          "expiryDate": {
-            "type": "string",
-            "description": "Expiry date, open if null",
-            "format": "date",
-            "example": "2022-08-12"
-          },
-          "comment": {
-            "maxLength": 240,
-            "minLength": 0,
-            "type": "string",
-            "description": "Comment",
-            "example": "Some comment"
-          }
-        },
-        "description": "Appointment information"
-      },
-      "NonAssociationIdResponse": {
-        "required": [
-          "offenderNo1",
-          "offenderNo2"
-        ],
-        "type": "object",
-        "properties": {
-          "offenderNo1": {
-            "type": "string",
-            "description": "The 1st offender"
-          },
-          "offenderNo2": {
-            "type": "string",
-            "description": "The 2nd offender"
-          }
-        },
-        "description": "Non association id"
-      },
-      "PageNonAssociationIdResponse": {
-        "type": "object",
-        "properties": {
-          "totalPages": {
-            "type": "integer",

```

Comment:
Did you forget to push this through jq as per ther readme? Doesn't really matter I suppose but it keeps it human readable `curl https://nomis-prisoner-api-dev.prison.service.justice.gov.uk/v3/api-docs | jq . > openapi-specs/nomis-sync-api-docs.json`

Reply:
OK, point taken - which I will forget the next time I do this :-)

Reply:
we could have a "refresh-all" script to all at the same time